gdal/swig/python/samples/README

```diff
@@ -4,115 +4,136 @@ Python interface may be used in your applications.
-                        classification of an image.  This is only an example
-                        that has stuff hardcoded.
+classify.py
+    Demonstrates using numpy for simple range based classification of an image.
+    This is only an example that has stuff hardcoded.
 
-gdal_lut.py             Read a LUT from a text file, and apply it to an image.
-                        Sort of a '1 band' version of pct2rgb.py.
+gdal_lut.py
+    Read a LUT from a text file, and apply it to an image.
+    Sort of a '1 band' version of pct2rgb.py.
 
-magphase.py             Example script computing magnitude and phase images
-                        from a complex image.
+magphase.py
+    Example script computing magnitude and phase images from a complex image.
 
-hsv_merge.py            Merge greyscale image into RGB image as intensity
-                        in HSV space.
+hsv_merge.py
+    Merge greyscale image into RGB image as intensity in HSV space.
 
-val_at_coord.py         Outputs the value of the raster bands at a given
-                        (longitude, latitude) or (X, Y) location.
+val_at_coord.py
+    Outputs the value of the raster bands at a given (longitude, latitude) or (X, Y) location.
 
-gdal_ls.py              Display the list of files in a virtual directory,
-                        like /vsicurl or /vsizip
+gdal_ls.py
+    Display the list of files in a virtual directory, like /vsicurl or /vsizip
 
-gdal_cp.py              Copy a virtual file
+gdal_cp.py
+    Copy a virtual file
 
-crs2crs2grid.py 	A script to produce PROJ.4 grid shift files from HTDP program.
-			http://trac.osgeo.org/proj/wiki/HTDPGrids
+crs2crs2grid.py
+    A script to produce PROJ.4 grid shift files from HTDP program.
+	http://trac.osgeo.org/proj/wiki/HTDPGrids
 
-ogrupdate.py            Update a target datasource with the features of a source datasource. Contrary to ogr2ogr,
-                        this script tries to match features between the datasources,
-                        to decide whether to create a new feature, or to update an existing one.
+ogrupdate.py
+    Update a target datasource with the features of a source datasource. Contrary to ogr2ogr,
+    this script tries to match features between the datasources,
+    to decide whether to create a new feature, or to update an existing one.
 
-ogr_layer_algebra.py    Application for executing OGR layer algebra operations.
+ogr_layer_algebra.py
+    Application for executing OGR layer algebra operations.
 
-gdalpythonserver.py     A Python script that can be used as the value of GDAL_API_PROXY_SERVER config. option
-                        Redirects on GDAL implementation, but could be used to implement a Python GDAL driver.
+gdalpythonserver.py
+    A Python script that can be used as the value of GDAL_API_PROXY_SERVER config.
+    option Redirects on GDAL implementation, but could be used to implement a Python GDAL driver.
```

Comment:
94 characters on the line - do we have some preferred limit?

Reply:
I'd be surprised if the standard should be <100 these days...
We all have wide screens I guess.